ckanext/string_to_location/controller.py

```diff
@@ -45,9 +48,9 @@ def map_location(self, id, resource_id):
 
         source_entity_type = OnsEntityTypes.LOCAL_AUTHORITY_DISTRICT
         target_entity_type = OnsEntityTypes.LOCAL_AUTHORITY_DISTRICT
-        resource_path = '/var/lib/ckan/resources/' + \
-            resource['id'][:3] + '/' + \
-            resource['id'][3:6] + '/' + resource['id'][6:]
+
+        resource_upload = uploader.get_resource_uploader(resource)
```

Comment:
🎨 `resource_uploader` looks like an interim variable which is only used on the line below. Consider merging the two statements to reduce the number of variables in play

Reply:
I feel like that might impact readability a little, would make it quite a long and slightly convoluted statement.

Reply:
You can format the statement to make the chaining more readable:

```python
resource_upload = uploader\
                                  .get_resource_uploader(resource)\
                                  .get_path(resource['id'])
```

or (perhaps more readable):

```python
resource_upload = (
                uploader.get_resource_uploader(resource)\
                               .get_path(resource['id'])
                                )
```
... with whatever whitespace feels right...

I tend to only create variables when they have a purpose, e.g. if that interim variable is used in feedback later on. Reducing the number of variable names in play increases the clarity of the code overall, though I'm not well-versed in Python idioms for this.

Reply:
^ the indentation on that is obviously borked, not sure why GH formats it like that